models/public/yolact-resnet50-fpn-pytorch/model.yml

```diff
@@ -0,0 +1,112 @@
+# Copyright (c) 2020 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+description: >-
+  YOLACT ResNet 50 is a simple, fully convolutional model for real-time instance segmentation
+  described in "YOLACT: Real-time Instance Segmentation" paper <https://arxiv.org/abs/1904.02689>.
+  Model pretrained in Pytorch* on COCO dataset. For details, see the repository <https://github.com/dbolya/yolact>.
+task_type: instance_segmentation
+
+files:
+  - name: yolact.py
```

Comment:
I think you should place all source files in a subdirectory, to avoid clutter in the download directory.

Reply:
fixed